src/env-inl.h

```diff
@@ -356,6 +356,25 @@ inline IsolateData* Environment::isolate_data() const {
   return isolate_data_;
 }
 
+inline void Environment::context_created(v8_inspector::V8ContextInfo info) {
+  contexts()->push_back(info);
+  if (inspector_agent()->IsStarted()) {
+    inspector_agent()->ContextCreated(info);
```

Comment:
@eugeneo the way the inspector_agent is setup, it is unsafe to call prior to it being started since platform_ does not exist.